assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -1,26 +1,27 @@
-FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:{{latest-image-tag:biweekly.\d{6}.\d{1}}}
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu118-py310-torch201:latest
+
+# Install basic dependencies and pip
+RUN apt-get update
 
 WORKDIR /
 
 # support Deepspeed launcher requirement of passwordless ssh login
 RUN apt-get update && apt-get -y upgrade
-RUN apt-get install -y openssh-server openssh-client
-
-COPY requirements.txt .
-RUN pip install -r requirements.txt --no-cache-dir
-
-# List installed packages
-RUN pip list
 
-## Delete
-RUN rm requirements.txt
+COPY . .
+RUN pip install --no-cache-dir -e .
+RUN pip list    # show installed packages
 
 # Copy scoring file
-COPY score.py /var/mlflow_resources/mlflow_score_script.py
+COPY src/score.py /var/mlflow_resources/mlflow_score_script.py
 ENV AZUREML_ENTRY_SCRIPT="mlflow_score_script.py"
 ENV AML_APP_ROOT="/var/mlflow_resources"
 
-# Inference requirements
+# clean conda and pip caches
+RUN conda clean -y --all
+RUN rm -rf ~/.cache/pip
+
+#Inference requirements
 COPY --from=mcr.microsoft.com/azureml/o16n-base/python-assets:20230419.v1 /artifacts /var/
```

Comment:
Isn't there a newer image for O16n?

Reply:
I'll leave that for figuring out in the next iteration. I think we might be removing this altogether.